qiskit_nature/runtime/vqe_runtime_client.py

```diff
@@ -0,0 +1,382 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The Qiskit Nature VQE Quantum Program."""
```

Comment:
I think you should update this slightly too. Maybe including some of your explanations from the PR description.

Reply:
I updated the class docstring, since I felt that's more visible than the module docstring 👍🏻